models/repo.go

```diff
@@ -392,6 +392,19 @@ func (repo *Repository) UnitEnabled(tp UnitType) bool {
 	return false
 }
 
+// AnyUnitEnabled if this repository has the any of the given units enabled
+func (repo *Repository) AnyUnitEnabled(tps ...UnitType) bool {
+	repo.getUnits(x)
```

Comment:
Unchecked error returned by `repo.getUnits(x)`

Reply:
I'm keeping same behaviour as in CheckUnit :) I will see if without much of rewriting there is anything I can do about it

Reply:
@ethantkoenig as this needs to be used in templates so it can not return error so I don't see much I can do here and it is same as it is in `UnitEnabled` method already

Reply:
Can we at least log it?